src/main/java/seedu/address/ui/ParticipantListPanel.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.participant.Participant;
+
+public class ParticipantListPanel extends UiPart<Region> {
+    private static final String FXML = "ParticipantListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(PersonListPanel.class);
+
+    @FXML
+    private ListView<Participant> participantListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
+     */
+    public ParticipantListPanel(ObservableList<Participant> participantList) {
+        super(FXML);
+        participantListView.setItems(participantList);
+        participantListView.setCellFactory(listView -> new ParticipantListPanel.ParticipantListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
```

Comment:
```suggestion
     * Custom {@code ListCell} that displays the graphics of a {@code Participant} using a {@code PersonCard}.
```
Currently the implementation is still PersonCard, to be changed to ParticipantCard? (Or no change is fine i guess) in the future.